src/KubernetesClient/Yaml.cs

```diff
@@ -190,6 +192,27 @@ public static string SaveToString<T>(T value)
             return stringBuilder.ToString();
         }
 
+        private class YamlRenameCamelCaseConvention : INamingConvention
+        {
+            private readonly INamingConvention _camelCase = CamelCaseNamingConvention.Instance;
+
+            private readonly IDictionary<string, string> _rename = new Dictionary<string, string>
+            {
+                { "namespaceProperty", "namespace" },
```

Comment:
any reason for this renaming?
seems weird if my config was renamed after loaded

Reply:
Yes, sadly when using `IgnoreUnmatchedProperties`, the `namespace` property will not be matched anymore. If discovered this while testing. With this renaming strategy, the properties are matched (again) correctly.

Reply:
Do we understand why this happens? I'd rather not make changes w/o understanding the reason for it.